haiku-apps/feedgator/feedgator-2.1.b.recipe

```diff
@@ -0,0 +1,40 @@
+SUMMARY="Feed the Gator"
+DESCRIPTION="FeedGator is a feed/RSS aggregator. \
+It's a native python3 program that uses Haiku APIs through Haiku-PyAPI \
+wrapper and the feedparser module. \
+Check the settings to enable some features like automated feed checks \
+or launch minimized. \
+The code is released under MIT license."
+HOMEPAGE="https://github.com/tmtfx/feedgator"
+COPYRIGHT="2024 Fabio Tomat (TmTFx)"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/tmtfx/FeedGator/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="7b3f7711ab6c9c7318357bf155fb3f6a25b75dcc29ec4b2b2f1a1e336a7ca0d1"
+SOURCE_DIR="FeedGator-$portVersion"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="yes"
+
+PROVIDES="
+	feedgator = $portVersion
+	app:FeedGator = $portVersion
+	cmd:feedgator.py
+	"
+
```

Comment:
No emptly line between PROVIDES and REQUIRES

Reply:
See https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines for more info.